packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -123,6 +123,21 @@ $isPaddedMargin: 24px;
   @include margin-left(-16px);
 }
 
+.dropHint {
+  display: inline-block;
+  position: absolute;
+  bottom: 0;
+  top: 100;
+  overflow: hidden;
+  height: inherit;
+  width: 2px;
+  z-index:100;
+
+  &.isDropping {
+    background: rgb(82, 79, 79);
+  }
+}
```

Comment:
Is this according to the UX? Or is this just a placeholder for now?

Reply:
For now, just a place holder to see it getting updated while dragging.

Reply:
For now, just a placeholder

---
In reply to: [188871768](https://github.com/OfficeDev/office-ui-fabric-react/pull/4857#discussion_r188871768) [](ancestors = 188871768)